### PR TITLE
Tests for Snap Dock

### PR DESCRIPTION
**Pull request by aziz512**

### CODE REVIEW

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@aziz512). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

File: scripts/test.js [hunk 1]
```diff
@@ -27,10 +27,11 @@ const fail = err => {
 require('./utils/build')('test')
     .then(() => require('./serve'))
     .then(async () => {
-        port = await launch({manifestUrl: 'http://localhost:1337/test-app.json'})
+        port = await launch({manifestUrl: 'http://localhost:1337/test/app.json'});
+	console.log('Openfin running on port ' + port);
```
Comment (pbaize) [inline scripts/test.js]: add a space before this console.log

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@aziz512). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

File: test/main.test.ts [hunk 2]
```diff
@@ -3,36 +3,204 @@ import {test} from 'ava';
...
 
-test('basic test', async t => {
-    const fin = await getConnection();
+let appIdCount = 0;
+
+test.before(async () => {
+    fin = await getConnection();
+});
+test.beforeEach(async () => {
+    const getAppName = () => 'a' + appIdCount++;
```
Comment (pbaize) [inline test/main.test.ts]: lets stick to something like 'test-app' or something a bit more meaningful than just a0

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@aziz512). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@aziz512). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@aziz512). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

File: package.json [hunk 3]
```diff
@@ -39,6 +39,7 @@
     "webpack": "^4.3.0"
   },
   "dependencies": {
+    "meow": "^5.0.0",
```
Comment (pbaize) [inline package.json]: Where is meow being used? Should it be a dev dependency?

**Pull request merged**